esp32_s3_korvo_2/include/bsp/esp32_s3_korvo_2.h

```diff
@@ -99,11 +98,50 @@ typedef enum {
     BSP_BUTTON_SET,
     BSP_BUTTON_VOLDOWN,
     BSP_BUTTON_VOLUP,
-    BSP_BUTTON_NUM,
     BSP_BUTTON_MAIN,
+    BSP_BUTTON_NUM,
 } bsp_button_t;
```

Comment:
I think, the we should split button indexes and button (GPIO/touch) numbers. Because, there can be situation, that the GPIO numbers and touch numbers will be same for two buttons.

Reply:
Good point. Could we use `bsp_button_t` as button index and define GPIO numbers in separate #define macros?

Reply:
Yes, it will be good. Define GPIOs same as pins for LCD.